controllers/reconciler.go

```diff
@@ -132,6 +133,14 @@ func (r *SingleClusterReconciler) Reconcile() (ctrl.Result, error) {
 		return reconcile.Result{}, err
 	}
 
+	// revert migrate-fill-delay to original value if it was set to 0 during scale down
+	// Passing first rack from the list as all the racks will have same migrate-fill-delay
+	if res := r.setMigrateFillDelay(r.getClientPolicyFromSpec(), r.aeroCluster.Spec.RackConfig.Racks[0],
```

Comment:
Instead of rack, send config only, as there is no other use of rack object in function definition.

Reply:
Done